build.gradle

```diff
@@ -19,6 +19,18 @@ plugins {
   id 'idea' // Support for IntelliJ IDEA projects
 }
 apply plugin: 'kotlin'
+apply plugin: 'jacoco'
+jacoco {
+    toolVersion = "0.7.6.201602180812"
+    reportsDir = file("build/customJacocoReportDir")
+}
+jacocoTestReport {
+    reports {
+        xml.enabled false
+        csv.enabled false
+        html.destination "build/jacocoHtml"
```

Comment:
No giving the reports specification will export an HTML report at ` build/reports/jacoco/test/html`. Guess we can leave it like this